webrtc.html

```diff
@@ -720,7 +720,7 @@
                   </li>
 
                   <li>
-                    <p>Fire a simple event named <code><a href=
+                    <p>Queue a task to fire a simple event named <code><a href=
```

Comment:
It looks like there is no objection to @jan-ivar's comment.   Meanwhile, the Mozilla team has confirmed offline that there is no concern from browser implementation perspective.  I will remove this specific change from the proposal.